canonical-web-and-design/maas-ui
# Change hardcoded paths to variables

## Done
Changes hardcoded paths including `/MAAS`, `/#` and `/r` to use environment variables. Fixes #422.

## QA
- Go to a pod details page and compose a machine
- Click the link which reads `More about spaces` and see that the link works as expected
- Login and out and make sure that works as expected
